docs/Dockerfile

```diff
@@ -13,7 +13,6 @@ RUN apt-get update && \
 
 COPY entrypoint.sh /
 
-EXPOSE 4000
```

Comment:
Why remove this port expose? Running the docker image and using the Jekyll Web server is sometimes useful.

Reply:
Well, jekyll web view is limited to viewing only one version of the docs. I'll update the docs generation manual later.